lean_packages/manifest.json

```diff
@@ -0,0 +1,24 @@
+{"version": 2,
+ "packages":
+ [{"url": "https://github.com/mhuisi/lean4-cli",
+   "rev": "112b35fc348a4a18d2111ac2c6586163330b4941",
+   "name": "Cli"},
+  {"url": "https://github.com/hargonix/LeanInk",
+   "rev": "439303af06465824588a486f5f9c023ca69979f3",
+   "name": "leanInk"},
+  {"url": "https://github.com/leanprover/doc-gen4",
+   "rev": "a2345380e861332fabecc61591cb274e61163a97",
+   "name": "doc-gen4",
+   "inputRev": "main"},
+  {"url": "https://github.com/xubaiw/Unicode.lean",
+   "rev": "25b32ef69fd6d8ae34300d51dc0b9403ffb1fd29",
+   "name": "Unicode"},
+  {"url": "https://github.com/leanprover-community/mathlib4.git",
```

Comment:
Why does doc-gen4 depend on mathlib4?

Reply:
doc-gen itself does not depend on mathlib for sure. If I grep for mathlib in a doc-gen build I get:
```
λ grep -r -i "mathlib" . | grep lakefile.lean
grep: ./lean_packages/mathlib/.git/index: binary file matches
grep: ./lean_packages/mathlib/.git/objects/pack/pack-0a95fa21965d4516ca298f75aa1f0b2a838b9b47.pack: binary file matches
grep: ./lean_packages/mathlib/.git/objects/pack/pack-f021064ed325132390895a54799a5c0197b0ef8a.pack: binary file matches
grep: ./lean_packages/mathlib/build/lib/Mathlib/Util/IncludeStr.olean: binary file matches
grep: ./lean_packages/mathlib/build/ir/Mathlib/Util/IncludeStr.o: binary file matches
./lean_packages/mathlib/lakefile.lean:package mathlib
./lean_packages/mathlib/lakefile.lean:lean_lib Mathlib
grep: ./lean_packages/lake/.git/objects/pack/pack-aa696e402774de9cdf8ccf888450b9d59cfe48af.pack: binary file matches
grep: ./lean_packages/lake/build/lib/Lake/CLI/Help.olean: binary file matches
grep: ./lean_packages/lake/build/lib/Lake/CLI/Init.olean: binary file matches
grep: ./lean_packages/lake/build/ir/Lake/CLI/Help.o: binary file matches
grep: ./lean_packages/lake/build/ir/Lake/CLI/Init.o: binary file matches
grep: ./build/lib/DocGen4/Output/Template.olean: binary file matches
./lean_packages/leanInk/test/dep/lakefile.lean:require mathlib from git "https://github.com/leanprover-community/mathlib4.git"
grep: ./build/bin/doc-gen4: binary file matches
grep: ./build/ir/DocGen4/Output/Template.o: binary file matches
```
so it would seem to me that `LeanInk` is the culprit here but also only with a test? Either way the `build` directory of mathlib4 inside of `lean_packages` of a doc-gen4 build is empty so it is definitely not a hard requirement for building the binary.

Reply:
I think this is a bug with `lake`.  Apparently it won't delete old manifest entries.  If I do `rm lean_packages/manifest.json && lake update` in `doc-gen4`, then the `mathlib4` entry goes away.